portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,15 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+// function addRandomGreeting() {
```

Comment:
Same. You may want to remove the commented code in next changelist. It's a good practice to always keep code clean.